UPGRADE-4.2.md

```diff
@@ -373,6 +373,7 @@ Validator
  * The `checkMX` and `checkHost` options of the `Email` constraint are deprecated
  * The component is now decoupled from `symfony/translation` and uses `Symfony\Contracts\Translation\TranslatorInterface` instead
  * The `ValidatorBuilderInterface` has been deprecated and `ValidatorBuilder` made final
+ * The `ValidatorTypeGuesser` can now guess the type of accepted files and out it into the FileType
```

Comment:
no need to change the UPGRADING file - but the CHANGELOG *of the component* should have that line (under 4.3.0)